support/generate_release_assets_redirect.sh

```diff
@@ -56,13 +56,10 @@ done < <(find "${SCRATCH_DIR}/src/metaschema" \
         -a ! -name '*metadata*' \
     -exec basename {} _metaschema.xml ';')
 
-# For each metaschema, append the appropriate schema and converter assets
-ASSETS=(
-    "${ROOT_METASCHEMAS[@]/%/_schema.json}"
-    "${ROOT_METASCHEMAS[@]/%/_schema.xsd}"
-    "${ROOT_METASCHEMAS[@]/%/_json-to-xml-converter.xsl}"
-    "${ROOT_METASCHEMAS[@]/%/_xml-to-json-converter.xsl}"
-)
+ASSETS=()
+while IFS='' read -r asset; do
+  ASSETS+=("$asset")
+done < <(make -C "${SCRATCH_DIR}" list-release-artifacts)
```

Comment:
Minor, non-blocking: should we not be consistent with the names given the Make target?

```suggestion
ARTIFACTS=()
while IFS='' read -r asset; do
  ARTIFACTS+=("$artifact")
done < <(make -C "${SCRATCH_DIR}" list-release-artifacts)
```

Reply:
Good point! I'm not sure why GitHub adopted such a confusing set of terms (release asset vs. CI artifact)